gst/edge/edge_src.c

```diff
@@ -369,6 +396,26 @@ gst_edgesrc_start (GstBaseSrc * basesrc)
   return TRUE;
 }
 
+/**
+ * @brief Stop edgesrc, called when state changed ready to null
+ */
+static gboolean
+gst_edgesrc_stop (GstBaseSrc * basesrc)
+{
+  GstEdgeSrc *self = GST_EDGESRC (basesrc);
+  int ret;
+
+  ret = nns_edge_stop (self->edge_h);
+
+  if (NNS_EDGE_ERROR_NONE != ret) {
+    nns_loge ("Failed to stop edgesrc. error code(%d)", ret);
+    return FALSE;
+  }
+
+  self->playing = FALSE;
```

Comment:
Thank you.
I don't know the condition of the stop well, so I have a question. If stop fails, should the self->playing value remain true?

Reply:
Thank you for the suggestion.
Even if stop fails, playing should be changed to `FALSE`.
Updated.